### ISSUE TITLE
About using multiple images

### ISSUE THREAD

**Issue by lzt02** [paulpanwang/POPE#12]
In your work, it was mentioned that better results can be achieved by inputting **multiple images**. I would like to evaluate multiple images as input, **how to proceed**.

Thanks in advance for your help!

**Comment by werdnabae** [comment]
Yes, I would like to do this too. Did you find a solution to do this task?
